### ISSUE TITLE
"Send to AltText.ai for Alt Text Generations when saving with no alt text" doesn't work

### ISSUE THREAD

**Issue by darylknight** [dispositiontools/craft-alt-text-generator#15]
If you enable this option in the plugin settings, and save an asset - it is not sent to alttext.ai and doesn't get any alt text added:

![image](https://github.com/user-attachments/assets/6599a2ce-fb58-49d1-b49c-55e5de62acde)

The integration in general is set up and working; we managed to submit all assets without alt text and they all got generated fine, but it isn't doing it for new assets we've uploaded and save when the Alt Text field is blank.

**Comment by dispositiontools** [comment]
thank you for this @darylknight. 
I'll get this sorted. I'm working on this tomorrow and working on bringing in a pull request. 
What version of Craft are you using it with? Craft 4 or Craft 5

**Comment by darylknight** [comment]
Thanks :) It's a Craft 5 site.

**Comment by dispositiontools** [comment]
I've been looking in to this. Are you using a custom field for the alt text?

**Comment by darylknight** [comment]
No, just the native alt field
